src/main/java/xyz/nucleoid/plasmid/mixin/game/event/ProjectileEntityMixin.java

```diff
@@ -0,0 +1,45 @@
+package xyz.nucleoid.plasmid.mixin.game.event;
+
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.EntityType;
+import net.minecraft.entity.projectile.ProjectileEntity;
+import net.minecraft.util.ActionResult;
+import net.minecraft.util.hit.BlockHitResult;
+import net.minecraft.util.hit.EntityHitResult;
+import net.minecraft.util.hit.HitResult;
+import net.minecraft.world.World;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+import xyz.nucleoid.plasmid.game.GameWorld;
+import xyz.nucleoid.plasmid.game.event.BlockHitListener;
+import xyz.nucleoid.plasmid.game.event.EntityHitListener;
+
+@Mixin({ ProjectileEntity.class })
```

Comment:
oh oops- I forgot to remove the {} here, this is a note so i remember to fix that :P